lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_retrospectives.py

```diff
@@ -340,7 +344,7 @@ def discharge_disposition(redcap_record: dict) -> Optional[str]:
     standardized_disposition = standardize_whitespace(disposition.lower())
 
     if standardized_disposition not in mapper:
-        raise Exception(f"Unknown discharge disposition value «{standardized_disposition}».")
+        raise UnknownHospitalDischargeDisposition(f"Unknown discharge disposition value «{standardized_disposition}» for barcode «{redcap_record['barcode']}».")
```

Comment:
Can we wrap this line to keep it from exceeding 100 characters?

Reply:
Fixed!